specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/appconfiguration.json

```diff
@@ -1617,11 +1617,11 @@
           }
         },
         "composition_type": {
-          "description": "The composition type describes how the key-values within the snapshot are composed. The 'all' composition type includes all key-values. The 'group_by_key' composition type ensures there are no two key-values containing the same key.",
+          "description": "The composition type describes how the key-values within the snapshot are composed. The 'key' composition type ensures there are no two key-values containing the same key. The 'key_label' composition type ensures there are no two key-values containing the same key and label.",
           "type": "string",
           "enum": [
-            "all",
-            "group_by_key"
+            "key_label",
+            "key"
```

Comment:
Does it matter to put `key` (the default) as the first item?

Reply:
Good idea. I'll update it.

Reply:
Order doesn't matter but alphabetical can be beneficial for reviews.

Reply:
My expectation was that generated SDKs would generate enums in the same order, leaving the first element to be the default value of the enum.

If that's not the case, then alphabetical may be better after all.

Reply:
SDKs do not use or care about the order in swagger. Some languages, like Go, do not even support the concept of enums. Also, there is no concept of a default. In SDKs that support enums, they are usually models as string values as enum numbers do not (and should not) go over the wire.